src/lib/chips/chip-list.spec.ts

```diff
@@ -242,3 +243,19 @@ class StaticChipList {
   chipDeselect(index: Number) {
   }
 }
+
+@Component({
+  template: `
+    <md-input-container>
```

Comment:
I do want to support this sort of thing in the future (the `mdInput` directive would go on the `md-chip-list` though and `md-chips-list` would have to provide some input-like interface for `mdInput` to work with). For now though what about something like this:

```html
<md-input-container>
  <md-chip-list md-prefix>
    <md-chip>Chip 1</md-chip>
    ...
  </md-chip-list>
  <input mdInput>
</md-input-container>
```